src/pages/LogIn.jsx

```diff
@@ -1,65 +1,147 @@
-import {useRef, useState, useEffect} from 'react';
+import { useRef, useState, useEffect } from 'react';
+import axiosInstance from '../api/axios';
 
 function LogIn() {
   const userRef = useRef();
   const errRef = useRef();
 
-  const [user, setUser] = useState('');
-  const [pwd, setPwd] = useState('');
+  const EMAIL_REGEX = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+  const PWD_REGEX = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%]).{8,24}$/;
+
+  const [email, setEmail] = useState('');
+  const [validEmail, setValidEmail] = useState(false);
+  const [password, setPassword] = useState('');
+  const [validPassword, setValidPassword] = useState(false);
```

Comment:
We don't need this `validPassword` value - see comment above

Reply:
Got you. Removed